packages/shopify-app-remix/src/server/shopify-app.ts

```diff
@@ -93,6 +97,17 @@ export function shopifyApp<
   return shopify as ShopifyApp<Config>;
 }
 
+function unauthenticatedAdminContextFactory<
```

Comment:
Wondering if it's too much to create a top-level `unauthenticated` folder for this kind of thing, since we're planning on adding more of them. We might want to punt extracting it until we actually add those methods, though.